src/components/layout.module.css

```diff
@@ -4,7 +4,7 @@
   flex-direction: column;
   min-height: 100vh;
   /* base styles; these could be kept in global styles */
-  font-family: system-ui, sans-serif;
+  font-family: "Inter", system-ui, sans-serif;
```

Comment:
Not for this PR, but I'd like to take a pass at how the starter's root layout can use the same CSS custom properties for the entire site